oak_runtime/src/node/http/tests.rs

```diff
@@ -45,22 +57,95 @@ impl crate::node::Node for OakNodeForTest {
         let invocation_receiver = Receiver::<HttpInvocation>::new(ReadHandle { handle });
 
         while let Ok(invocation) = invocation_receiver.receive(&runtime) {
+            // Compare the request and response labels. If the echo-node (which gets the
+            // request-label) can write to the response-channel, let the `EchoNode` send the
+            // response. Otherwise, the RouterNode should send the response to the caller.
+            let request_label =
+                get_label(&runtime, invocation.clone().receiver.unwrap().handle.handle).unwrap();
```

Comment:
Reading the label from the receiver object seems more idiomatic than doing it via low-level handles.

Edit: actually I'm not sure whether that's implemented natively, or just in the SDK.